insights/parsers/xfs_quota.py

```diff
@@ -68,14 +68,15 @@ class XFSQuotaState(CommandParser):
         <class 'insights.parsers.xfs_quota.XFSQuotaState'>
         >>> len(quota_state.user_quota)
         3
-        >>> quota_state.user_quota[0] == {'device': '/dev/sdd', 'accounting': 'ON', 'enforcement': 'ON', 'inode': '#131 (1 blocks, 1 extents)', 'blocks_grace_time': '7 days', 'blocks_max_warnings': '5', 'inodes_grace_time': '7 days', 'inodes_max_warnings': '5', 'realtime_blocks_grace_time': '7 days'}
+        >>> quota_state.user_quota['/dev/sdd'] == {'device': '/dev/sdd', 'accounting': 'ON', 'enforcement': 'ON', 'inode': '#131 (1 blocks, 1 extents)', 'blocks_grace_time': '7 days', 'blocks_max_warnings': '5', 'inodes_grace_time': '7 days', 'inodes_max_warnings': '5', 'realtime_blocks_grace_time': '7 days'}
         True
     """
 
     def parse_content(self, content):
-        self.user_quota = []
-        self.group_quota = []
+        self.user_quota = {}
+        self.group_quota = {}
```

Comment:
Hi @TZ3070 ,  
I am good with the data structure changes here. 
Just a tiny update is missed here - for the description and data type of docstrings `Attributes` (line#24)  in docstrings.